components/humantask/humantask-explorer-web/src/web/controller/statsController.jag

```diff
@@ -0,0 +1,36 @@
+<%
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
The license header years were added in the proper way. Fixed with commit b3f1319eb19fd419fbe953e4af3ff3cfd484d00e